### DISCUSSION TITLE
Regarding Running and compilation on Visual Studio Mac

### DISCUSSION THREAD

**Discussion by pirhana97** [danielmonettelli/netmaui-chatgpt-app-challenge#1] [Q&A]
Hello Core developers!

Thanks for making your idea about introducing ChatGPT on mobile app platforms open source. I believe we can learn and do a lot once it is accessible on mobile phones. I came across your project while searching for making contribution to any backend/infrastructure related changes in C# language. 

I understand from going through the document, the app is in semi-stable state, and I was wondering if compiling and running the codebase on Visual Studio on Mac is something which is supported (eg. to see it as preview?) which would give me an understanding of the working more along with code reading and debugging. If possible, could you brief about how to run the application?

Could you also let me know if comments are something which is appreciated or prevented in the code base in future? Because, generally, I have observed adding summary when adding a new interface with service contracts gives developers a better understanding of the code workflow.

Thanks and appreciate for taking out the time to read :)

**Comment by danielmonettelli** [comment]
Hello @pirhana97! Thank you very much for your interest in collaborating with our project. Regarding your first question, .NET MAUI is supported in both Visual Studio 2022 on Windows and VS for Mac. I recommend watching Daniel Hindrikes' video starting at minute 5 to learn how to run .NET MAUI projects in VS for Mac. Skip the setup section of the video tutorial and focus on running the application.

I want to mention that Bryan and I have collaborated on a pull request in Luis's original repository, as a token of appreciation for his valuable help to the .NET MAUI community. Currently, I have created a collaborative project derived from the initial project. Feel free to comment on the code base. I appreciate your support and look forward to your collaboration in the future.

- [ Inspiration project made by Luis Beltrán ](https://github.com/icebeam7/NetMauiGPTApp)
- [ .NET MAUI - Official Documentation ](https://dotnet.microsoft.com/en-us/apps/maui)
- [ Daniel Hindrikes -> .NET MAUI - How to use with Mac ](https://www.youtube.com/watch?v=4DmpY9looao)
